files/meethyde/update.json

```diff
@@ -0,0 +1,9 @@
+{
+  "packageManager": "github",
+  "name": "cdn",
+  "repo": "meethyde/meethyde",
+  "files": {
+		"basePath": "release/",
```

Comment:
There is no folder `release` here https://github.com/MeetHyde/MeetHyde/tree/v0.1
The bot will fail